frontend/pages/invite/[club]/[invite]/[token]/index.js

```diff
@@ -82,6 +83,7 @@ class Invite extends Component {
             className="has-text-centered"
             style={{ margin: 30, marginTop: 60 }}
           >
+            <ClubMetadata club={club} />
```

Comment:
This won't work as you expect, as this case only happens if the club does not exist. It would be better to use `<Metadata />` in this case.